reference/shipping.v3.yml

```diff
@@ -142,6 +143,65 @@ paths:
                     CA: '508313'
                     AU: '817355'
                     ALL: '501000'
+
+  '/shipping/settings':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+
+    get:
+      operationId: getCheckoutSettings
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/checkoutSettings'
+                  meta:
+                    $ref: '#/components/schemas/metaCollection'
+      description: |-
+        Get checkout settings.
+
+        ```http
+        GET /shipping/settings
```

Comment:
It'd be great you removed this example in the description because it'll be redundant when it's live on the site

![shipping settings](https://github.com/user-attachments/assets/6886eb37-e02e-403c-933f-e2b53a0f35a8)

Reply:
Thanks Andrea, I've removed this.🍻